### ISSUE TITLE
How much concurrency support ?

### ISSUE THREAD

**Issue by rfyiamcool** [msgpack-rpc/msgpack-rpc-python#10]
How much concurrency support?

When you use block modules, then  rpc server meet block?

**Comment by repeatedly** [comment]
Yes. You can use AsyncResult with your any concurrency module in the server.
